src/api/get-user.js

```diff
@@ -0,0 +1,12 @@
+const getUser = async (octokit) => {
+    try {
+        const {
+            data: { login },
+          } = await octokit.rest.users.getAuthenticated();
+          return login;
+    } catch(e){
+        return `Error: ${e.message}`;
```

Comment:
I wouldn't use the same path for success and errors.  If you return a string, it worked.  If you throw, it failed.  Keep errors as exceptions so you can tell success from failure.